### DISCUSSION TITLE
allow navigation through explorer settings patcher properties window using keyboard.

### DISCUSSION THREAD

**Discussion by fernando-jose-silva** [valinet/ExplorerPatcher#628] [Ideas]
Hope everything is ok with everyone

First I would like to apologize, this text is translated, I am Brazilian and I speak only Portuguese.
I would also like to apologize, I posted this text as a problem, after I filled the request to be posted as a discussion. 

I am visually impaired and I use this application, as well as all windows applications using a screen reader, in this case the screen reader
novda
www.nvaccess.org
I met explorer patcher and from the beginning I liked it a lot, my big congratulations.
With this software I can modify my windows and I feel the difference when navigating using the keyboard, other customization tools are bad, many times they only modify the appearance, but those who use the computer don't use a keyboard and don't notice any difference.
Sorry to elaborate, but I would like to give an example of how useful your work is.
I installed the explorer patcher for my blind students, to recapture points in windows 11 that they really liked, like showing the desktop button when tabbing the desktop, always showing all items in the toolbar, always showing all the programs in the start menu., and everyone is very grateful.
Now comes my request if possible.
When I go to configure the app explorer patcher I enter into properties, I can't navigate with tab and arrows in the window.
So I'm almost unable to customize the explorer patcher, which is a very big deficiency.
To customize the software I have to use the character recognition ocr tool of the screen reader nvda, as well as the mouse click simulation, which for someone who is blind and uses only the keyboard is a very bad experience.
I would like to ask you please if possible to implement a graphical interface that allows configuring the explorer patcher app using the keyboard, and implementing accessibility tools so that it is accessible to screen readers.
Sorry, I can't help anymore, I'm not a programmer, I'm a basic computer teacher for visually impaired students.
I also recommend the github
from the nvda screen reader project
https://github.com/nvaccess/nvda/
I believe that there you can learn more about the screen reader and find out more information on how to make the explorer patcher more accessible to screen readers, so you would talk among programmers, laugh.
Also your knowledge I'm sure would make the screen reader nvda evolve and help more visually impaired in the world.
Sorry for the big message, congratulations on the work, and you are helping with accessibility and ease of use for the visually impaired, congratulations.

**Comment by valinet** [comment]
I have posted a reply in #627.
